sdcm/nemesis.py

```diff
@@ -4319,6 +4325,106 @@ def disrupt_add_remove_mv(self):
         finally:
             self.unset_current_running_nemesis(cql_query_executor_node)
 
+    def change_audit_config(self, audit_config: dict):
```

Comment:
why this check is unique to nemesis ? why can't it be part of cluster implementation ?

also what's the point of changing the yaml, and then checking the yaml changed ? are we testing SCT implementation ?